docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -687,6 +684,134 @@ If Alluxio has already been deployed with helm and now you want to enable FUSE,
 $ helm upgrade alluxio -f config.yaml --set fuse.enabled=true --set fuse.clientEnabled=true alluxio-charts/alluxio
 ```
 
+### Short-circuit Access
+
+Short-circuit access enables clients to perform read and write operations directly against the
+worker bypassing the networking interface.
+For performance-critical applications it is recommended to enable short-circuit operations
+against Alluxio because it can increase a client's read and write throughput when co-located with
+an Alluxio worker.
+
+#### Properties to Enable Short-Circuit Operations
+
+This feature is enabled by default, however requires extra configuration to work properly in
+Kubernetes environments.
+See sections below for how to configure short-circuit access.
+
+#### Disable Short-Circuit Operations
+To disable short-circuit operations, the operation depends on how you deploy Alluxio.
+
+> Note: As mentioned, disabling short-circuit access for Alluxio workers will result in 
+worse throughput 
+
+***Using `helm`***
+You can disable short circuit by setting the properties as below:
+
+```properties
+shortCircuit:
+  enabled: false
+```
+
+***Using `kubectl`***
+You should set the property `alluxio.user.short.circuit.enabled` to `false` in your `ALLUXIO_WORKER_JAVA_OPTS`.
+
+```properties
+-Dalluxio.user.short.circuit.enabled=false
+```
+
+You should also manually remove the volume `alluxio-domain` from `volumes` of the Pod definition 
+and `volumeMounts` of each container if existing.
+
+#### Short-Circuit Modes
+There are 2 modes for using short-circuit.
+
+##### `local`
+In this mode, the Alluxio client and local Alluxio worker recognize each other if the client hostname 
+matches the worker hostname.
+This is called *Hostname Introspection*.
+In this mode, the Alluxio client and local Alluxio worker share the tiered storage of Alluxio worker.
+
+***Using `helm`***
+You can use `local` policy by setting the properties as below:
+
+```properties
+shortCircuit:
+  enabled: true
+  policy: "local"
+```
+
+***Using `kubectl`***
+In your `alluxio-configmap.yaml` you should add the following properties to `ALLUXIO_WORKER_JAVA_OPTS`:
+
+```properties
+-Dalluxio.user.short.circuit.enabled=true -Dalluxio.worker.data.server.domain.socket.address= -Dalluxio.worker.data.server.domain.socket.as.uuid=false
```

Comment:
Not 100% sure it should be `alluxio.worker.data.server.domain.socket.address=`

Reply:
you can ask user to remove this property altogether (unset)